validator/client/service.go

```diff
@@ -285,6 +302,29 @@ func ConstructDialOptions(
 	return dialOpts
 }
 
+// Reloads the validating keys every set interval to check if they have changed and updates
```

Comment:
This comment seems wrong - it's not reloading at a set interval, it's watching for changes